src/topology/quasi_separated.lean

```diff
@@ -0,0 +1,116 @@
+/-
+Copyright (c) 2022 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import topology.subset_properties
+import topology.sets.compacts
+
+/-!
+# Quasi-separated spaces
+
+A topological space is quasi-separated if the intersections of any pairs of compact open subsets
+are still compact.
+Notible examples include spectral spaces and Hausdorff spaces.
```

Comment:
Could you please mention an example of a **non** quasi-separated space too?

Reply:
```suggestion
Notable examples include spectral spaces and Hausdorff spaces.
```

Reply:
Noetherian spaces are all examples and you should add that as an instance :)
More generally, is_noetherian -> is_quasi_separated